kmapper/drawing.py

```diff
@@ -41,6 +41,13 @@ def draw_matplotlib(g, ax=None, fig=None, layout="kk"):
 
     """
     import networkx as nx
+    import os
+    # https://stackoverflow.com/a/50089385/5917194
+    import matplotlib as mpl
+    if os.environ.get('DISPLAY','') == '':
```

Comment:
this is necessary to get tox to run the tests. `.travis.yml` has something similar for handingly $DISPLAY